.circleci/config.yml

```diff
@@ -11,7 +11,18 @@ jobs:
           name: Install shellcheck
           command: sudo apt install shellcheck
       - run:
-          name: Show version information
+          name: Install Go
+          command: |
+            cd /tmp
```

Comment:
interesting way to install in /tmp

Reply:
Yeah, otherwise I ran into permissioning problems. I thought this is good enough.